realm/realm-library/src/main/java/io/realm/OrderedRealmCollectionSnapshot.java

```diff
@@ -48,7 +49,7 @@
  * }
  * </pre>
  */
-public class OrderedRealmCollectionSnapshot<E> extends OrderedRealmCollectionImpl<E> {
+public class OrderedRealmCollectionSnapshot<E> extends OrderedRealmCollectionImpl<E> implements Freezable<OrderedRealmCollection<E>> {
```

Comment:
Shouldn't this also result in `@Override` being added to the relevant methods?

Reply:
Nope, because this class already had a `freeze` method. It already had `@Override` because it came from `OrderedRealmCollectionImpl`, but now it comes from `Freezable`.